src/components/Home.jsx

```diff
@@ -19,48 +17,50 @@ const Home = () => {
         }
     };
 
-useEffect(() => {
-    fetchArticles();
-}, []);
+    useEffect(() => {
+        fetchArticles();
+    }, []);
 
-useEffect(() => {
-    if (articles.length > 0) {
-        let sortedArticles = [...articles];
-        if (sortOption === "date") {
-            sortedArticles.sort((a, b) => new Date(b.created_at) - new Date(a.created_at));
-        } else if (sortOption === "title") {
-            sortedArticles.sort((a, b) => a.title.localeCompare(b.title));
-        } else if (sortOption === "author") {
-            sortedArticles.sort((a, b) => a.author.localeCompare(b.author));
+    useEffect(() => {
+        if (articles.length > 0) {
+            let sortedArray = [...articles];  
+            if (sortOption === "date") {
+                sortedArray.sort((a, b) => new Date(b.created_at) - new Date(a.created_at));
```

Comment:
If you've done the query sorting part on your backend, it'd better to make use of your api rather than sort it on frontend